bin.js

```diff
@@ -122,7 +122,7 @@ function compile () {
       messages[name].dependencies.forEach(function (e, i, enc) {
         var name = encodings.name(e)
         if (name) name = 'encodings.' + name
-        else name = e.name
+        else name = 'encodings.varint'
```

Comment:
In fact enums are encoders and need to be added too

Reply:
did you mean to close this? :)

Reply:
Mhh I actually am not sure :laughing: I made some tests and it looks like now it's working but I can't decode -_-.